src/index.ts

```diff
@@ -1550,7 +1574,8 @@ export const formats: IFormat[] = [
   hexChecksumChain('GO_LEGACY', 6060),
   bech32mChain('XCH', 8444, 'xch', 90),
   getConfig('NULS', 8964, nulsAddressEncoder, nulsAddressDecoder),
-  bech32Chain('AVAX', 9000, 'avax'),
+  getConfig('AVAX', 9000, makeBech32Encoder('avax'), makeAvaxDecoder('avax')),
+  getConfig('AVAXC', 9005, makeChecksummedHexEncoder(), makeAvaxcDecoder()),
```

Comment:
Can you verify only C-prefix is allowed? 
Also, please use ChainID for EVM compatible chains instead of SLIP44 cointype? Please see https://docs.ens.domains/ens-improvement-proposals/ensip-11-evmchain-address-resolution for more detail. You should be able to make use of `convertEVMChainIdToCoinType(chainId)` which I explained at https://medium.com/the-ethereum-name-service/improved-ens-support-for-evm-compatible-chains-19fc86e9722b (but probably have to strip C- off/back in when storing)